www/js/toolbar.js

```diff
@@ -45,18 +48,64 @@ const toggleOverlayUI = (toolbarItem, show) => {
   });
 };
 
+const setListeners = () => {
+  if (window.socket !== undefined) {
+    window.socket.on('data', data => {
+      const { search } = global.core.search;
+      const isPinCorrect = parseInt(data.pin, 10) === adminPin;
+
+      const loadShabad = (shabadId, verseId, gurmukhi) => {
```

Comment:
add comment here for why we need gurmukhi.

// We need gurmukhi here to add for history support. Yes, we can get the gurmukhi from desktop side as well, but I didn't want to mess with current history code. As that could lead to a lot of bugs.. which is already a minefield because of documented state management issues with current code.